doc/man1/x509.pod

```diff
@@ -479,7 +479,8 @@ When a certificate is created set its subject name to the given value.
 The arg must be formatted as I</type0=value0/type1=value1/type2=...>.
 Keyword characters may be escaped by \ (backslash), and whitespace is retained.
 Empty values are permitted, but the corresponding type will not be included
-in the request.
+in the certificate. Giving a single I</> will lead to an empty sequence of RDNs
+(a NULL subject DN).
```

Comment:
I don't think that "NULL" is the right word here. It is present, but empty.  For EE certificates, this is actually recommended by PKIX, since the subject information is supposed to come from the subject alternative names. And for CAs like _Let's Encrypt_, which set the subject name to just `/CN=hostname.example`, this becomes necessary when the FQDN length exceeds 64 bytes (X.509 CN length limit).

Reply:
A "NULL DN" is the X.500 special term for exactly what you mean: an empty sequence of RDNs.
See for instance https://tools.ietf.org/html/rfc4210#appendix-D.1:
> A "special" X.500 DN is called the "NULL-DN"; this means a DN
       containing a zero-length SEQUENCE OF RelativeDistinguishedNames
       (its DER encoding is then '3000'H).

Reply:
OK, thanks, sorry about the noise.